ecs-init/docker/docker_unspecified.go

```diff
@@ -34,11 +34,7 @@ func getPlatformSpecificEnvVariables() map[string]string {
 // It mounts dhclient executable, leases and pid file directories when built
```

Comment:
Please update the adjacent comment to reflect the removal of dhclient.

Reply:
done.  I need to get into the habit of adding `-i` to my greps.